routes/movie.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express"); 
+const router = express.Router(); 
+const Movie = require("../moduls/movies");
+const User = require("../moduls/users");
+
+router.post("/", async (req, res) => {
+    try {
+        const movie = new Movie({
+            name: req.body.name, 
+            yera: req.body.yera,
+            type: req.body.type,
+            contry: req.body.userId
+        });
+        await movie.save();
+        const user = await User.findById({_id: movie.contry});
```

Comment:
Is this what we want? Do we want to try and find by ID where the movie contry? Perhaps it would be better to search by user id?